pkg/bgp/speaker/pod_cidr.go

```diff
@@ -97,20 +93,7 @@ func (s *MetalLBSpeaker) announcePodCIDRs(cidrs CidrSlice) error {
 	l.WithField("advertisements", cidrs).
 		Info("Advertising CIDRs to all available session")
 
-	var eg errgroup.Group
```

Comment:
the `withdraw` method also does this same thing with `Set`. should withdraw now clean `SvcAds["podcidr"]` instead?

also wonder if we should be using `SetBalancer` and `DeleteBalancer` instead...

Reply:
Yes, a Set() of an empty advertisements array tells MetalLB to "remove all sessions". It's noted somewhere in src.